app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    author = User.find(params['comment']['author_id'].to_i)
+    post = Post.find(params['comment']['post_id'].to_i)
+    text = params['comment']['text']
+    @comment = Comment.new(post: post, author: author, text: text)
```

Comment:
- [ ] A comment is created on behalf of the currently logged in user (current_user) of the app. Kindly make use of the `current_user` of the app to create a comment.